tmt/steps/provision/testcloud.py

```diff
@@ -301,7 +301,7 @@ def go(self) -> None:
                 self.info('disk', f"{value} GB", 'green')
             elif key == 'connection':
                 self.verbose('connection', value, 'green')
-            elif value is not None:
+            elif value:
```

Comment:
I'd probably recommend keeping `is None`, adding a special case `elif key == 'key'` instead, to handle the list nature of the value. You're probably going to encounter strange output in the case of `verbose()` call below, too: `value` parameter is expected to be a string, so passing `value` to `info()` or `verbose()` should be stopped by mypy sooner or later, and `str()` would produce ugly output`. Can you try `verbose('key', '\n'.join(self.key), 'green')`? Should give you better output (`info() & co. indent multi-line strings better) - I don't know whether there's a native support for "print value but nicely formatted when it's a list" operations.

Reply:
@psss @lukaszachy is there a canonical way how to use `info()` & co. to print out lists and dictionaries? IIUIC, `info()` and its underlying code like `tmt.utils.indent()` would, sooner or later, convert `value` to a string, creating an ugly output. Converting a list to string before that, to inject line breaks `indent()` understands seems like a way, but is it the best way?

Reply:
Other plugins convert list to string by [listed](https://fmf.readthedocs.io/en/stable/modules.html#fmf.utils.listed)  from fmf.utils (e.g. [install](https://github.com/teemtee/tmt/blob/main/tmt/steps/prepare/install.py#L110))

Reply:
Thanks @lukaszachy for review, right here `self.info('key', fmf.utils.listed(value), 'green')` is used.